task8/goodTweet/src/main/java/bsu/fpmi/twitter/Posts.java

```diff
@@ -0,0 +1,63 @@
+package bsu.fpmi.twitter;
+
+import java.util.List;
+
+public class Posts {
+    private List<Post> posts;
+
+    public Post getPost(String id){
+        for (int i = 0; i < posts.size(); i++) {
+            if (posts.get(i).getId().equals(id)) {
+                return posts.get(i);
+            }
+        }
+        return null;
+    }
+
+    public boolean validate(Post post){
+        if (post.getId() == null || post.getId().length() == 0)
```

Comment:
A more conventional form is `post.getId() == null || post.getId().isEmpty()`

Reply:
And, please, *never* use `if (...)` without curly braces. This is not prohibited by language itself, but this is never welcome in teamwork. They say, such block is harder to read, and it's prone to mistakes